callerid: document constraint of the anonymous mode (#703)
wazo-platform/wazo-platform.org: content/uc-doc/administration/callerid.md

rewrite the CallerID that is attached to the trunk, others will leave the CallerID untouched, some
operators will only rewrite the CallerID if you use an unauthorized CallerID, etc.

### Anonymous CallerID

If the user needs to do anonymous calls there are a few things to consider.

> The effect of choosing to do anonymous calls does the following.

Reviewer: ```suggestion
The effects of making anonymous calls are as follows:
```

> 1. It sets the `From` header to `“Anonymous” <sip:anonymous@anonymous.invalid>`
> 2. It adds the `Privacy` header with the value `id`
> 3. It adds the `P-Asserted-Identity` header with the value `tel:<Caller ID number>`

Reviewer: ```suggestion
1. The `From` header is set to `“Anonymous” <sip:anonymous@anonymous.invalid>`
2. The `Privacy` header is added with the value `id`
3. The `P-Asserted-Identity` header is added with the value `tel:<Caller ID number>`
```

This means that Wazo needs to be able to choose a number to set in the `P-Asserted-Identity` header
for the feature to work.

The prefered way to add a number to the `P-Asserted-Identity` header is to configure the caller ID
field in your outgoing call extension list. If the outgoing call uses multiple trunks it might not
be possible to select a caller ID for the extensions.

> If this is not possible you could also configure the `callerid` field of the `endpoint` section of
> your trunk.

Reviewer: ```suggestion
If this is not possible the `callerid` field of the `endpoint` section of
the trunk can also be used.
```

Finaly, the PJSIP `endpoint` options `send_pai` and `trust_id_outbound` can be set to `yes`. This

> will send internal information to the provider for all calls done using that trunk instead of a

Reviewer: ```suggestion
will send internal information to the provider for all calls made using that trunk instead of a
```
